lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -704,6 +705,9 @@ fromFlatSlot n = SlotId e (fromIntegral s)
 epochLength :: Integral a => a
 epochLength = 21600
 
+newtype SlotLength = SlotLength (Quantity "second/slot" Word8)
```

Comment:
I think the dimension is really `"second"` here. We'd rather say: "The slot length is 60s", than, "The slot length is 60s per slot"? Said differently, `slot` is really just a scalar I think.


Debatable. But what do you think?

Reply:
after thinking about "second" sounds more reasonable